docs/docs/sourcing-from-yaml.md

```diff
@@ -0,0 +1,232 @@
+    - item:
+        Bacon ipsum dolor amet short ribs brisket venison rump drumstick pig sausage
+        prosciutto chicken spare ribs salami picanha doner. Kevin capicola sausage,
+        buffalo bresaola venison turkey shoulder picanha ham pork tri-tip meatball meatloaf
+        ribeye. Doner spare ribs andouille bacon sausage. Ground round jerky brisket
+        pastrami shank.
+  links:
+    - to: "/page1"
+```
+
+What the code block above is doing, is nothing more, nothing less, than creating a simple YAML object in which:
+
+- Each `path` is a page's endpoint.
+- The `contents` list holds some data to be displayed.
+- The `links` list holds some endpoints to other pages.
+
+## Gatsby configuration
+
+Now that we have almost all the parts necessary in place, you'll need to make a small adjustment to your Gatsby website.
+
+Start by creating a `gatsby-node.js` with the following contents:
+
+```javascript
+const fs = require("fs")
+const yaml = require("js-yaml")
+exports.createPages = ({ actions }) => {
+  const { createPage } = actions
+  const ymlDoc = yaml.safeLoad(fs.readFileSync("./content/index.yaml", "utf-8"))
+  ymlDoc.forEach(element => {
+    createPage({
+      path: element.path,
+      component: require.resolve("./src/templates/basicTemplate.js"),
+      context: {
+        pageContent: element.content,
+        links: element.links,
+      },
+    })
+  })
+}
+```
+
+Breaking down the code into smaller parts:
+
+1. The `js-yaml` package installed earlier is imported.
+2. The `index.yaml` file is loaded and the content parsed.
+3. Through the use of Gatsby's `createPage()` api, the parsed file will be used to create some pages programmatically.
+4. Using the `context` property, will allow every piece of data you add there, to be accessible in the page as a special prop named `pageContext` allowing it to be consumed. You can read more about it [here](https://www.gatsbyjs.org/docs/creating-and-modifying-pages/)
+
+## Review of the steps so far
+
+Doing a quick review of what you have done so far:
+
+1. Created a new Gatsby website based on a template.
+2. Added some dependencies to handle YAML files.
+3. Created the file and folder structure to house the contents.
+4. Configured Gatsby to fetch the data and create some pages programmatically.
+
+## Gatsby Template
+
+The only missing thing is the template used for each page created.
+
+Create a file called `basicTemplate.js` in the `src/templates/` folder and add the following contents:
+
+```javascript
+import React from "react"
+import uuid from "uuid"
+import { Link } from "gatsby"
+const basicTemplate = props => {
+  const { pageContext } = props
+  const { pageContent, links } = pageContext
+
+  return (
+    <div style={{ maxWidth: `960px`, margin: `1.45rem` }}>
+      <div>
+        {pageContent.map(data => {
+          return <div key={uuid.v4()}>{data.item}</div>
+        })}
+      </div>
+      <div>
```

Comment:
Let's make this more semantic with an unordered list:
```suggestion
      <ul>
```

Reply:
changes applied to both the example and document